OpsCenter_6.8_Release_Notes.md

```diff
@@ -1,18 +1,29 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.36
+1 Apr 2024
+
+## Monitoring,UI
+* Fixed issue with installing datastax-agent. (OPSC-17503)
+
+## UI
+* Reverted the code changes made for limiting a session to a specific user agent and IP address. (OPSC-17278)
```

Comment:
I think this should be labeled as core rather than ui. The login can happen entirely via the api.